SOLID/S/DY/GoodSolution/Cup.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Solid\S\DY\GoodSolution;
+
+use Solid\S\DY\GoodSolution\Interfaces\StorableInterface;
+use Solid\S\DY\GoodSolution\Interfaces\StorageInterface;
+
+class Cup implements StorageInterface
+{
+    private $volume;
+    private $color;
+
+    /**
+     * Cup constructor.
+     * @param int $volume
+     * @param string $color
+     */
+    public function __construct(int $volume, string $color)
```

Comment:
In this case ( just 2 class properties and class seems finished) it is ok,  but when amount of properties is not finally defined, this solution became not that usable as it seems on first view